frontend/src/components/pages/CampCreation/index.tsx

```diff
@@ -59,18 +59,76 @@ const CampCreationPage = (): React.ReactElement => {
 
   let isCampDetailsFilled = false;
 
+  const useValidateTimeOrder = (
+    start: string,
+    end: string,
+    canBeEqual = false,
+  ): boolean => {
+    return useMemo(() => {
+      try {
+        const [startHours, startMinutes] = start
+          .split(":")
+          .map((num) => parseInt(num, 10));
+        const [endHours, endMinutes] = end
+          .split(":")
+          .map((num) => parseInt(num, 10));
+
+        if (Number.isNaN(startHours) || Number.isNaN(endHours)) {
+          return true;
+        }
+
+        if (canBeEqual) {
```

Comment:
Checked with Andy, EDLP is allowed to have the same times as the camp's start and end time, so I added this `canBeEqual` check.